cs41_final.sh

```diff
@@ -59,6 +83,91 @@ good_bye(){
   echo -e "${MENU}***************** ${RED_TEXT}GOOD BYE ${MENU}******************${NORMAL}"
   echo -e "${MENU}*********************************************${NORMAL}"
 }
+
+#----- Option 1 -----
+currentUsers(){
+  echo "Currently Logged-In Users:"
+  users
+}
+
+#----- Option 2 -----
+listActiveProcess(){
```

Comment:
the output for this is confusing. 

It uses option 1's list of users (see comments on option 1 above)
I typed 1, and it tried to find processes for user '1'